contributors/guide/README.md

```diff
@@ -234,6 +235,17 @@ When reviewing PRs from others [The Gentle Art of Patch Review](http://sage.thes
 
 Note: if your pull request isn't getting enough attention, you can use the [#pr-reviews](https://kubernetes.slack.com/messages/pr-reviews) channel on Slack to get help finding reviewers.
 
+## Contributions best practices
+
+- Do write clear and meaningful Git commit messages.
+- Make sure to include _Github Special Keywords_ which references the issue and automatically closes the issue when PR is merged.
```

Comment:
Would it be possible to add a note on `@` mentioning people in commit messages? We don't currently allow those.^^;;;
ref: https://github.com/kubernetes/test-infra/pull/9387

Reply:
Sure!

Reply:
@mrbobbytables Done!